tests/test_config.py

```diff
@@ -66,3 +66,29 @@ def test_ssl_config(certfile_and_keyfile):
     config.load()
 
     assert config.is_ssl is True
+
+
+async def asgi3app(scope, receive, send):
+    pass
+
+
+def asgi2app(scope):
+    async def asgi(receive, send):
+        raise RuntimeError("Something went wrong")
+
+    return asgi
```

Comment:
Should we move this alongside `asgi_app` and `wsgi_app`, as `asgi2_app`?

Reply:
ok